index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="If you need a software developer, I'm here for you, just see my work and contact me">
+        <title>Lucas' Portfolio</title>
+        <link href="style.css" rel="stylesheet">
+    </head>
+    <body>
+      <div class="container">
```

Comment:
- [ ] You wrapped all content with the `div` tag. The `div` tag is unnecessary here. There is no need. You can delete it.

Maybe you misunderstand this [practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Close%20all%20HTML,always%20needs%20%3C/div%3E). This means closing all HTML elements with closing tags.